clouddriver-core/src/main/java/com/netflix/spinnaker/clouddriver/config/AccountDefinitionConfiguration.java

```diff
@@ -53,6 +65,21 @@
 @Log4j2
 public class AccountDefinitionConfiguration {
 
+  @Bean
+  @ConditionalOnMissingBean
+  public AccountDefinitionAuthorizer accountDefinitionAuthorizer(
+      Optional<FiatPermissionEvaluator> maybePermissionEvaluator,
+      @Value("${services.fiat.enabled:false}") boolean isFiatEnabled) {
```

Comment:
I'd love to pass a configuration object here instead of `@Value` but I'm not sure such an object exists today...

Reply:
Well, there's the object where the evaluator comes from, but that wasn't enough of a hint here.